desktop/mac/index.md

```diff
@@ -227,9 +225,20 @@ For information about the reset options, see [Logs and Troubleshooting](troubles
 
 ### Software Updates
 
-The Software Updates section notifies you of any updates available to Docker Desktop. You can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.
+The **Software Updates** section notifies you of any updates available to Docker Desktop. When there's a new update, you can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.
+
+> **Disable automatic check for updates**
+>
+> Starting with Docker Desktop 4.2.0, the option to turn off the automatic check for updates is available for users on all Docker subscriptions, including Docker Personal. Update to [Docker Desktop 4.2.0](../mac/release-notes/index.md) to start using this feature.
```

Comment:
```suggestion
> Starting with Docker Desktop 4.2.0, the option to turn off the automatic check for updates is available for users on all Docker subscriptions, including Docker Personal and Docker Pro. Update to [Docker Desktop 4.2.0](../mac/release-notes/index.md) to start using this feature.
```

Reply:
It was also behind a pay wall for pro users so figure we might want to highlight!

Reply:
Thanks Steph. I've updated the text.